packages/http-client-java/emitter/src/code-model-builder.ts

```diff
@@ -289,7 +289,7 @@ export class CodeModelBuilder {
           },
           language: {
             default: {
-              serializedName: arg.name,
+              serializedName: arg.serializedName,
```

Comment:
Change for Qiaoqiao's case in scrum (serializedName should be serializedName).

@haolingdong-msft Is this correct?